Resources/Prototypes/_White/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -0,0 +1,739 @@
+      - 0 # Miasma
+      - 0 # N2O
+      - 0 # Frezon
+      - 1871.71051 # BZ
+      temperature: 293.15
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 600
+      behaviors:
+        - !type:DoActsBehavior
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)